test/common/index.js

```diff
@@ -431,20 +437,30 @@ function runCallChecks(exitCode) {
 
 
 exports.mustCall = function(fn, expected) {
```

Comment:
Perhaps change the parameter name to something like 'exact' to make the distinction clearer?